app/services/v1x0/catalog/add_to_order.rb

```diff
@@ -0,0 +1,30 @@
+module V1x0
+  module Catalog
```

Comment:
@syncrou Should we add a Service module here so we know that we are calling a service
icon = **Service**::Catalog::CreateIcon.new(icon_params).process.icon